app/policies/person_policy.rb

```diff
@@ -0,0 +1,5 @@
+class PersonPolicy < ApplicationPolicy
+  def admin?
+    person.present? && person.admin?
```

Comment:
This is some built in Pundit stuff

```
class ApplicationPolicy
  attr_reader :person, :record

  def initialize(person, record)
    @person = person
    @record = record
  end
#...
```

If we trace person through all the loops and hoops it is essentially `current_person`